piptools/scripts/sync.py

```diff
@@ -147,3 +147,69 @@ def cli(
             ask=ask,
         )
     )
+
+
+def _build_install_flags(
+    finder,
+    no_index=False,
+    index_url=None,
+    extra_index_url=None,
+    trusted_host=None,
+    find_links=None,
+    user_only=False,
+    cert=None,
+    client_cert=None,
+):
+    """
+    Builds install flags with the given finder and CLI options.
```

Comment:
According to PEP 257, you should use an imperative style sentence for a docstring title.
```suggestion
    Build install flags with the given finder and CLI options.
```
And maybe s/Build/Compose/ ?

Reply:
Good naming! Thanks.